psalm.xml

```diff
@@ -3,6 +3,8 @@
     errorLevel="2"
     phpVersion="8.2"
     resolveFromConfigFile="true"
+    findUnusedBaselineEntry="true"
```

Comment:
I need to set this flag to avoid a deprecation warning. It defaults to `false`, but I think enabling it will help us keep the baseline clean. 🤞🏻

Reply:
Yeah, we are the ones (well I) [asking for that feature](https://github.com/vimeo/psalm/issues/9102) so better use it at some point :stuck_out_tongue: